### PR TITLE
chore(features): Move permanent flags to their own file

### PR DESCRIPTION
**Pull request by evanpurkhiser**
- Sorts feature flags again

 - Moves flags that will never be removed to their own file to keep them
   logically separated from flags which are more "temporary" and may one
   day graduate to be permanent.

### CODE REVIEW

File: src/sentry/features/flags/permanent.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    manager.add("organizations:team-insights", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    # NOTE: Don't add features down here! Sort them alphabetically! The order
+    #       features are registered is not important.
+
+    manager.add("projects:custom-inbound-filters", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("projects:data-forwarding", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("projects:discard-groups", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("projects:rate-limits", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("projects:servicehooks", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
+    # NOTE: Don't add features down here! Sort them alphabetically! The order
```
Comment (JoshFerge) [inline src/sentry/features/flags/permanent.py:61]: asking engineers to do this without tooling feels burdensome.

Comment (evanpurkhiser) [reply src/sentry/features/flags/permanent.py:61]: Does your editor not have a line sort feature?

Comment (evanpurkhiser) [reply src/sentry/features/flags/permanent.py:61]: wow I sound grumpy

Comment (JoshFerge) [reply src/sentry/features/flags/permanent.py:61]: i don't know, i don't sort stuff 😭

**Comment by evanpurkhiser** [comment]
Need to have another go when there's less churn in this file

**Pull request closed without merging**